docs/aws/audit/route53monitoring/rules/domain_auto_renew_remediation.mdx

```diff
@@ -1,6 +1,203 @@
+3. In the Route 53 dashboard, select "Hosted zones" from the navigation pane on the left side of the screen.
+4. Select the domain name you want to check. This will bring up a list of all the records associated with that domain.
+5. Click on the "Domain Registration" tab. Here, you can see the details of your domain registration, including whether or not auto-renew is enabled. If "Auto Renew" is set to "On", then auto-renew is enabled. If it is set to "Off", then auto-renew is not enabled.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine and configure it with your AWS account credentials. You can do this by running the following commands:
+
+   Installation:
+   ```
+   pip install awscli
+   ```
+   Configuration:
+   ```
+   aws configure
+   ```
+   You will be prompted to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. List all the hosted zones: Once the AWS CLI is set up, you can list all the hosted zones in your AWS account by running the following command:
+   ```
+   aws route53 list-hosted-zones
+   ```
+   This command will return a list of all the hosted zones along with their details.
+
+3. Get the details of each hosted zone: For each hosted zone, you can get more details including the domain name, record set count, and more importantly, whether the auto-renew is enabled or not. Use the following command to get the details of a specific hosted zone:
+   ```
+   aws route53 get-hosted-zone --id <HostedZoneId>
+   ```
+   Replace `<HostedZoneId>` with the ID of the hosted zone you want to check. This command will return the details of the specified hosted zone.
+
+4. Check the Auto Renew status: In the output of the previous command, look for the `Config` attribute. If the `AutoRenew` attribute under `Config` is set to `true`, then auto-renew is enabled for that hosted zone. If it's set to `false`, then auto-renew is not enabled.
+
+Please note that AWS Route 53 does not directly support the auto-renew feature. The auto-renew feature is generally associated with domain registration services. AWS Route 53 does support domain registration, and when you register a domain, auto-renew is enabled by default. However, this setting can be changed in the Route 53 domain registration settings, not in the hosted zone settings.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can install it using pip:
+
+```python
+pip install boto3
+```
+
+2. Import the necessary modules and create a session using your AWS credentials:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>